crates/adkg/src/aba/crain20.rs

```diff
@@ -579,14 +579,13 @@ where
 
                 // Inform the parent thread through the channel once
                 // decide(v)
-                if let Some(sender) = sender.take() {
-                    if sender.send(est).is_err() {
+                if let Some(sender) = sender.take()
+                    && sender.send(est).is_err() {
```

Comment:
this'll probably fail, we're using rust 1.86 in the ci which does not support if-let-chains

Reply:
we can update the ci version tho, we'll need to do quite a few clippy fixes though, rust 1.88 became stricter